content/docs/user-guide/scripting/_index.md

```diff
@@ -7,8 +7,14 @@ weight: 900
 
 **Open 3D Engine (O3DE)** includes two scripting languages for creating logic and behaviors: Script Canvas and Lua.
 
-**Script Canvas** is a visual scripting environment. In the Script Canvas editor, you create, connect, and rearrange graphical nodes that provide a visual representation of the logic flow. Script Canvas offers an approachable environment to author behaviors using the same framework as Lua and C++. You can use Script Canvas to create scripts without coding experience.
+**Script Canvas** is a general purpose, visual scripting environment. In the Script Canvas editor, you lay out and connect graphical nodes that provide a visual representation of the logic flow. Script Canvas offers an approachable environment to author behaviors using the same framework as Lua and C++. You can use Script Canvas to create scripts without coding experience.
```

Comment:
Some pages in this PR are inconsistent on whether to capitalize "editor" or not. If Script Canvas Editor is a proper name, capitalize "Editor". Also, consider bolding the first instance of it (per https://o3de.org/docs/contributing/to-docs/style-guide/format/#bold-proper-application-names). 

```suggestion
**Script Canvas** is a general purpose, visual scripting environment. In the Script Canvas Editor, you lay out and connect graphical nodes that provide a visual representation of the logic flow. Script Canvas offers an approachable environment to author behaviors using the same framework as Lua and C++. You can use Script Canvas to create scripts without coding experience.
```

Reply:
Good call out. Will do.